modules/openapi-generator/src/main/resources/typescript-angular/provide-api.mustache

```diff
@@ -0,0 +1,14 @@
+import { makeEnvironmentProviders } from "@angular/core";
+import { {{configurationClassName}}, {{configurationParametersInterfaceName}} } from './configuration';
+import { BASE_PATH } from './variables';
+
+export function provideApi(configOrBasePath: string | {{configurationParametersInterfaceName}}) {
```

Comment:
please add the return type [EnvironmentProviders](https://angular.dev/api/core/EnvironmentProviders);